src/shogun/mathematics/graph/ops/shogun/Input.h

```diff
@@ -16,14 +16,19 @@ namespace shogun {
 			return "Input";
 		}
 
+		void evaluate() override
```

Comment:
@vigsterkr let me know if you can think of a better way to feed data to the graph.
Basically the input node needs to be fed data, which comes from SGContainer. So it has to be different from the other nodes which can just grab data from the previous node